spec/requests/api/v1/items/item_api_find_all_spec.rb

```diff
@@ -0,0 +1,125 @@
+require 'rails_helper'
+
+describe "Search Items using find_all and parameters" do
+  before do
+    create_list(:item, 3)
+    @item = Item.last
+  end
+
+
+  it "returns an item based on the id" do
+
+    get "/api/v1/items/find_all?id=#{@item.id}"
+    expect(response).to be_success
+
+    items = JSON.parse(response.body, symbolize_names: true)
+
+    expect(items[0][:id]).to eq(3)
```

Comment:
Could maybe refactor this by saving item to a variable and testing off of that, but that'll be a for later if we want :)